gensim/test/test_sklearn_integration.py

```diff
@@ -67,5 +68,15 @@ def testCSRMatrixConversion(self):
             self.assertTrue(isinstance(v, six.string_types))
             self.assertTrue(isinstance(k, int))
 
+    def testPipline(self):
+        model = SklearnWrapperLdaModel(id2word=dictionary, num_topics=2, passes=100, minimum_probability=0, random_state=numpy.random.seed(0))
+        text_lda = Pipeline([('model', model)])
```

Comment:
Can a pipeline contain two things? From lda to logistic regression would be good. Also could you please add it to the tutorial.